third_party/2and3/flask/globals.pyi

```diff
@@ -0,0 +1,15 @@
+# Stubs for flask.globals (Python 3.6)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .app import Flask
+from .wrappers import Request
+from typing import Any
+from werkzeug.local import LocalStack
+
+_request_ctx_stack: LocalStack
+_app_ctx_stack: LocalStack
+current_app: Flask
+request: Request
```

Comment:
There is an error on `flask.request.access_route[-1]` complaining that `access_route` (inherited as `Callable[[], Any]` from `werkzeug`) is not indexable. Maybe it should be a property in `werkzeug`?

Reply:
Indeed. Fixed.